### ISSUE TITLE
Restrict keel to specific namespaces

### ISSUE THREAD

**Issue by xoss** [keel-hq/keel#464]
Hello keel team,

is it possible to restrict keel to a specific namespace. 

Following scenario:
We have a namespace called 'dev', which not only holds all deployments and services needed to build up our app, but also the required configs in configMaps and secrets.

Once we are sure that a current setup is stable, we take a snapshot of this entire namespace by creating a new namespace (let's name it 'androgynous-antelope', yes we loosely follow the ubuntu naming scheme) and clone the entire dev namespace into that new namespace. After that, we move the staging ingresses from our old staging namespace (let's call it 'zarking-zebu') to the new stating namespace ('androgynous-antelope'). Once this namespace turns out to be stable, we can also move the prod ingresses from our old prod namespace to the new namespace. This way, we are even able to quickly roll back (just by moving the ingress back to the old namespace, if the new namespace eventually turns out to be not stable).

Long story short: Once we took the snapshot and cloned the 'dev' namespace into a new staging namespace, we don't want keel to do automatic releases on this new namespace. So, in the end, we would like to configure keel to just do continuous deployments in namespace 'dev'. 

Is sth. like this configurable?

Thanks for your work and I'm looking forward to your reply.

**Comment by hlihhovac** [comment]
+1

**Comment by KacperGawdzinski** [comment]
would it be possible to expand this solution for multiple namespaces, not only one?
